api/src/cron/cron.ts

```diff
@@ -2,7 +2,7 @@ import cron from 'node-cron';
 
 const cronTab = [
   {
-    interval: '1-25/5 0 0 * * 5',
+    interval: '*/5 0,1,2,3,4 * * 5',
```

Comment:
If my math is correct, this would be a total of 100 runs, so a total of 2000 users with jobs.